connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -546,6 +547,34 @@ message StatFreqItems {
   optional double support = 3;
 }
 
+
+// Returns a stratified sample without replacement based on the fraction
+// given on each stratum.
```

Comment:
`It will invoke 'Dataset.stat.sampleBy' (same as 'StatFunctions.sampleBy') to compute the results.` should be added.

Reply:
It would be better to keep the comment

Reply:
nice, will update